proto/service.proto

```diff
@@ -62,6 +62,30 @@ service API {
     };
   }
 
+  rpc MultiTransferByCustomer(MultiTransferBeginCustomerRequest) returns (TransferStatusResponse){
+    option (google.api.http) = {
+      post: "/v1/multi-transfer/customer"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Multi transfer cross chanel be customer";
```

Comment:
be or by?

Reply:
fixed. It was copied from other endpoints descriptions. Fixed them also